tests/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
+project(models_test)
+
+enable_testing()
+
+set(MODEL_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/)
+include_directories("${CMAKE_CURRENT_SOURCE_DIR}/../")
+
+add_executable(models_test
+               dataloader_tests.cpp
+               utils_tests.cpp
+)
+
+# Link dependencies of test executable.
+target_link_libraries(models_test
+  ${COMPILER_SUPPORT_LIBRARIES}
+  ${ARMADILLO_LIBRARIES}
+  ${BOOST_LIBRARIES}
+  ${Boost_FILESYSTEM_LIBRARY}
+  ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY}
```

Comment:
Hi @kartikdutt18 do we have to include all boost libraries here again after including `BOOST_LIBRARIES`?

Reply:
Yes, that was needed. It doesn't work without them.

Reply:
We can clear the list of boost libraries [here](https://github.com/mlpack/models/blob/de580de972993e54f4ff6f7a195ce384f33f5eb7/CMakeLists.txt#L183). It can be handled by visual studio itself.